Quantum-Control-Applications/Superconducting/Single Fixed Transmon/configuration.py

```diff
@@ -80,6 +80,9 @@ def readout_macro(threshold=None, state=None, I=None, Q=None):
 square_pi_len = 100
 square_pi_amp = 0.1
 
+drag_coef = 0
+anharmonicity = 0
```

Comment:
I think it is more intuitive , and also in the case for the `py-qua-tools` it should be changed from `delta` to `anaharmonicity` what do you think?

Reply:
Yea we can do it.
Note that it is considered a breaking API change, because if someone used delta=xxx it works break...
We can solve it by accepting a kwarg argument, and if it's delta, support it but add a deprecated message

Reply:
I see super interesting, first time I face a breaking API change --- not sure what is best. Maybe what makes us work the least xD -- we can just give `delta=anahamornicity` as is listed in the configuration now

Reply:
It's not hard to do, this is how it would look like:
![image](https://user-images.githubusercontent.com/53370976/179393782-79fe82b1-e969-4712-b31b-6fbe3d02aff5.png)

We just need to decide :) 
WDYT? Kevin? Theo?

Reply:
I completely agree, we can keep this depreciation message for the next versions and remove it later